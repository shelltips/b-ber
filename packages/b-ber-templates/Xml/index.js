<<<<<<< HEAD
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _classCallCheck2 = require('babel-runtime/helpers/classCallCheck');

var _classCallCheck3 = _interopRequireDefault(_classCallCheck2);

var _createClass2 = require('babel-runtime/helpers/createClass');

var _createClass3 = _interopRequireDefault(_createClass2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Xml = function () {
    function Xml() {
        (0, _classCallCheck3.default)(this, Xml);
    }

    (0, _createClass3.default)(Xml, null, [{
        key: 'container',
        value: function container() {
            return '<?xml version="1.0"?>\n            <container version="1.0" xmlns="urn:oasis:names:tc:opendocument:xmlns:container">\n                <rootfiles>\n                    <rootfile full-path="OPS/content.opf" media-type="application/oebps-package+xml"/>\n                </rootfiles>\n            </container>\n        ';
        }
    }, {
        key: 'mimetype',
        value: function mimetype() {
            return 'application/epub+zip';
        }
    }]);
    return Xml;
}();

exports.default = Xml;
||||||| parent of 832bbc7... Restructuring
=======
"use strict";

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _classCallCheck2 = require("babel-runtime/helpers/classCallCheck");

var _classCallCheck3 = _interopRequireDefault(_classCallCheck2);

var _createClass2 = require("babel-runtime/helpers/createClass");

var _createClass3 = _interopRequireDefault(_createClass2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Xml = function () {
    function Xml() {
        (0, _classCallCheck3.default)(this, Xml);
    }

    (0, _createClass3.default)(Xml, null, [{
        key: "container",
        value: function container() {
            return "\n            <?xml version=\"1.0\"?>\n            <container version=\"1.0\" xmlns=\"urn:oasis:names:tc:opendocument:xmlns:container\">\n                <rootfiles>\n                    <rootfile full-path=\"OPS/content.opf\" media-type=\"application/oebps-package+xml\"/>\n                </rootfiles>\n            </container>\n        ";
        }
    }]);
    return Xml;
}();

exports.default = Xml;
>>>>>>> 832bbc7... Restructuring
