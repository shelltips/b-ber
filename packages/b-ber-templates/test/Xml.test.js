import Xml from '../src/Xml'

describe('templates.Xml', () => {
    test('container matches the snapshot', () => {
        expect(Xml.container()).toMatchSnapshot()
    })

    test('mimetype matches the snapshot', () => {
        expect(Xml.mimetype()).toMatchSnapshot()
    })
})
