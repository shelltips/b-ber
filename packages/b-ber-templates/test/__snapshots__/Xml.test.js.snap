// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`templates.Xml container matches the snapshot 1`] = `
"<?xml version=\\"1.0\\"?>
            <container version=\\"1.0\\" xmlns=\\"urn:oasis:names:tc:opendocument:xmlns:container\\">
                <rootfiles>
                    <rootfile full-path=\\"OPS/content.opf\\" media-type=\\"application/oebps-package+xml\\"/>
                </rootfiles>
            </container>
        "
`;

exports[`templates.Xml mimetype matches the snapshot 1`] = `"application/epub+zip"`;
